### PR TITLE
Clean up conditional-content.md

### PR DESCRIPTION
**Pull request by skaterdav85**

### CODE REVIEW

File: guides/release/components/conditional-content.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ is passed in and is truthy.
 <aside ...attributes>
   <div
     class="avatar {{if @isActive "is-active"}}"
-    title="{{@title}}"
+    title={{@title}}
```
Comment (rwjblue) [inline guides/release/components/conditional-content.md:108]: I'd probably still keep these quotes personally. I'm curious, what made you remove them?

Comment (skaterdav85) [reply guides/release/components/conditional-content.md:108]: Wouldn't this fail `no-unnecessary-concat`?

**Comment by jenweber** [comment]
It looks like the recommended default was changed in the fall to exclude `no-unnecessary-concat`:  https://github.com/ember-template-lint/ember-template-lint/pull/899/files 

In general I think we mostly want our code samples to only cover the recommended set, so I wouldn't want to go update every example. However it's fairly harmless either way, leaning towards being beneficial for older apps, so I'll merge this.

**Comment by jenweber** [comment]
P.S. Thanks for helping out with cleaning up some guides content! If you find anything else that seems amiss, let us know!

**Comment by skaterdav85** [comment]
@jenweber Interesting, I missed this update. Why would it be recommended for older apps and not newer apps?

**Comment by jenweber** [comment]
I’m not sure, but perhaps there is an issue or PR on it. I did notice there is a “stylistic” set of configs too, so maybe it has something to do with a shift towards that.

**Comment by skaterdav85** [comment]
I'm also curious if this is just a stylistic lint rule or if it has any performance implications (although maybe negligible). I know when I first came across this rule a long time ago, I thought that it maybe existed because doing `attribute="{{foo}}" would result in an extra concatenation operation, as the name of the rule suggests.

**Pull request merged**